clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -601,7 +601,8 @@ template <typename Container>
 static auto toJSONStrings(llvm::json::OStream &JOS, Container &&Strings) {
   return [&JOS, Strings = std::forward<Container>(Strings)] {
     for (StringRef Str : Strings)
-      JOS.value(Str);
+      if (!Str.ends_with("SDKSettings.json"))
```

Comment:
We should add a brief comment here (and to the other places you're skipping this output) to explain why we have this downstream change.